modules/swiss/src/main/SwissApi.scala

```diff
@@ -315,11 +315,10 @@ final class SwissApi(
     }
 
   private def getGameRanks(swiss: Swiss, game: Game): Fu[Option[GameRanks]] =
-    game.whitePlayer.userId.ifTrue(swiss.isStarted) so { whiteId =>
-      game.blackPlayer.userId.so: blackId =>
+    swiss.isStarted.so:
```

Comment:
please verify this refactor, if I understand correctly, this new implement looks more in sync with the logic:
if swiss.isStarted and both players hasId
then query ranking
then get ranking for each player byId
then return GameRanks.